lib/tessel/commands.js

```diff
@@ -5,7 +5,7 @@ module.exports.scanWiFi = function() {
   return ['ubus', 'call', 'iwinfo', 'scan', '{"device":"wlan0"}'];
 };
 module.exports.stopRunningScript = function() {
-  return ['/etc/init.d/tessel-app', 'stop'];
+  return ['killall', 'node'];
```

Comment:
Before merging, add this as an issue so that we know it'll be a problem for later.